omod/src/main/resources/apps/aijar_homepage_app.json

```diff
@@ -76,5 +76,20 @@
     "config": {
       "registrationAppLink": "registrationapp/registerPatient.page?appId=aijar.registrationapp.registerPatient"
     }
-  }
+  },{
```

Comment:
@slubwama How about the settings that are prefixed as ugandaemr?

Reply:
What does it have to do with this line.

Reply:
@slubwama This may have sifted with other commits but it has to do with the line 89 - admin/maintenance/settings.list?show=Aijar